packages/shared/package.json

```diff
@@ -15,17 +15,17 @@
   },
   "dependencies": {
     "@eten-tech-foundation/scripture-utilities": "workspace:*",
-    "@lexical/mark": "^0.33.0",
-    "@lexical/utils": "^0.33.0",
+    "@lexical/mark": "^0.33.1",
+    "@lexical/utils": "^0.33.1",
     "@sillsdev/scripture": "^2.0.2",
     "epitelete": "^0.2.20",
-    "lexical": "^0.33.0",
+    "lexical": "^0.33.1",
     "open-patcher": "1.0.0-beta.1"
   },
   "devDependencies": {
     "json-difference": "^1.16.1",
-    "proskomma-core": "^0.10.10",
-    "proskomma-json-tools": "^0.8.21"
+    "proskomma-core": "^0.11.0",
+    "proskomma-json-tools": "^0.9.0"
```

Comment:
I have not been upgrading these because I don't know that Abel wants that. If you have checked with him then by all means go ahead.